service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java

```diff
@@ -83,14 +81,12 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
 
         caseDetails.getData().putAll(applicationsFeeCalculator.calculateFee(caseData));
 
-        if (featureToggleService.isServeOrdersAndDocsToOthersEnabled() && caseData.hasRespondentsOrOthers()) {
-            caseDetails.getData().put("hasRespondentsOrOthers", "Yes");
-            caseDetails.getData().put("people_label", peopleInCaseService.buildPeopleInCaseLabel(
-                caseData.getAllRespondents(), caseData.getOthers()));
+        caseDetails.getData().put("hasRespondentsOrOthers", "Yes");
```

Comment:
why has the `caseData.hasRespondentsOrOthers` been removed?

Reply:
I'll put it back, I think it was removed while resolving the merge conflicts